pkg/machineproviders/providers/openshift/machine/v1beta1/providerconfig/vsphere.go

```diff
@@ -90,16 +92,26 @@ func (v VSphereProviderConfig) InjectFailureDomain(fd machinev1.VSphereFailureDo
 	newVSphereProviderConfig.providerConfig.Workspace = newVSphereProviderConfig.getWorkspaceFromFailureDomain(failureDomain)
 	topology := failureDomain.Topology
 
+	logNetworkInfo(newVSphereProviderConfig.providerConfig.Network, "CPMS Network before FD: %v", v.logger)
```

Comment:
Our logs are somewhat consistently using full names for things like this, in the way that docs would spell it out, please rephrase as 
```suggestion
	logNetworkInfo(newVSphereProviderConfig.providerConfig.Network, "control plane machine set network before failure domains: %v", v.logger)
```

Reply:
Sure, I'll update this.